src/ts/helper/AdHelper.ts

```diff
@@ -78,29 +107,75 @@ export class AdHelper {
     const adInfo: Conviva.ConvivaMetadata = {
       'c3.ad.id': ad.id,
       'c3.ad.technology': Conviva.Constants.AdType.CLIENT_SIDE,
-      'c3.ad.position': adPosition,
+      'c3.ad.position': AdHelper.mapClientSideAdPosition(adBreakEvent.adBreak, player),
       'c3.ad.system': adSystemName,
       'c3.ad.creativeId': creativeId,
       'c3.ad.firstAdId': firstAdId,
-      'c3.ad.mediaFileApiFramework': mediaFileApiFramework,
-      'c3.ad.firstAdSystem': firstAdSystem,
-      'c3.ad.firstCreativeId': firstCreativeId,
+      [Conviva.Constants.ASSET_NAME]: adTitle,
+      [Conviva.Constants.STREAM_URL]: ad.mediaFileUrl || 'NA',
+
+      // TODO This is not exposed currently. Add it whenever the player
+      // exposes it. Related discussion https://bitmovin.slack.com/archives/C0LJ16JBS/p1716801970037469.
+      'c3.ad.mediaFileApiFramework': 'NA',
+
+      // TODO these two are not exposed currently. Add them whenever the player
+      // exposes them similar to https://github.com/bitmovin-engineering/player-android/pull/3147.
+      // Related discussion https://bitmovin.slack.com/archives/C0LJ16JBS/p1716801796326889.
+      'c3.ad.firstAdSystem': 'NA',
+      'c3.ad.firstCreativeId': 'NA',
```

Comment:
Not sure if we really add internal links to the public repo. Usually we try to avoid that.

Reply:
These links were in the repo already from the previous PR - https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/104#discussion_r1625652063.

Reply:
Okay 🙈